server/services/prismic.js

```diff
@@ -184,7 +189,7 @@ export async function getCuratedList(id: string) {
 }
 
 function createExhibitionPromos(allResults: Object): Array<ExhibitionPromo> {
-  const allPromos = allResults.results.map((e):ExhibitionPromo => {
+  return allResults.map((e):ExhibitionPromo => {
```

Comment:
Wonder if we should implement this eslint rule for consistency? https://github.com/gajus/eslint-plugin-flowtype/blob/master/.README/rules/space-before-type-colon.md

Reply:
Could whack it in another PR?